### DISCUSSION TITLE
Templates Repository?

### DISCUSSION THREAD

**Discussion by semi710** [eylles/pywal16#46] [Q&A] [answered]
Hey do you think it would be a great idea to have a pywal template's repository with documentations explaining how this can be utilized with other software?

coz if i see i have 
![image](https://github.com/eylles/pywal16/assets/60490474/8bff3a37-309e-4ac5-b2d8-71615b803007)
this much templates that i have modified and use.

**Comment by MyGitHubBlueberry** [comment]
Agreed, this would be great.
But for now only [wiki](https://github.com/eylles/pywal16/wiki/User-Template-Files) is available.

From what I can see, this repo is still in work and readme and documentation almost not touched. But looking forward I think template repo might be done and readme and wiki will have better look and contain up-to-date information :)

**Reply by eylles** [reply]
well, the readme and wiki don't really need heavy updating, all things considered the changes from pywal 3.3.0 haven't been so heavy or at least it doesn't seem like that to me.

then again when i first proceeded with the fork i did so for my own use, that is why for the first releases i didn't even bother changing the package's name nor uploading to pypi as i figured "meh if someone cares enough they'll do that", it was later that i decided "eh may as well do" and didn't think anything of it, it was later that i had something of a change of mind, not a big one mind you, still i'm maintaining pywal16 primarly for my own use as i'm not a fan of carefully choosing a colorscheme to match my mood or whatever and then finding wallpapers that fit said colorscheme, i'm more of the mind of "let's see what pywal spits back with this image, if i like it i'll use it for a season" and that's how i roll...

**Comment by eylles** [comment]
yeh, been thinking of adding a template repo but dunno what would be a good way to proceed with it tbh, as not everything i do with pywal16 is templates, perhaps something more like the https://codeberg.org/nsxiv/nsxiv-extra repo where it is an assortment of stuff, so that it can also contain snippets and small scripts that don't merit their own repos.

**Comment by eylles** [comment, marked as answer]
i think everyone will be pleased to know i've finally gotten around and organized the [pywal-extra](https://github.com/eylles/pywal-extra) repo for templates, scripts, functions and other stuff, will mention it on the pywal16 readme and probably on the customization section of the wiki.

**Reply by eylles** [reply]
forgot to tag @niksingh710 @MyGitHubBlueberry

**Reply by semi710** [reply]
Nice to see a repository where many templates can reside.
rn m travelling else would have created a pr.

But if someone can add a few of my templates from my [repo](https://github.com/niksingh710/gdots/tree/master/.config/wal/templates). Or I will do it later.

**Reply by eylles** [reply]
just took a look and saw that you got a kde colorscheme template, i'm thinking of borrowing it for another repo to be able to build a kde colorscheme from a gradience json, also still have to make [pywal16-libadwaita/build-kvantum.](https://github.com/eylles/pywal16-libadwaita/blob/master/scripts/build-kvantum.py) into a real thing that works since pywal templates are rather limited for the needs of kvantum svg themes.
